azerothcore/azerothcore-wotlk
# fix(DB/loot): Dalaran Miner and Brewmaster drops

Dalaran Miner and Brewmaster were dropping certain items related to their
professions with 100% probability. This makes sense thematically, but according 
to classic wowhead drop tables, this is not right.

The report mentions only the Brewmaster, but after returning to the area I found the
Miner's loot table to also have guaranteed drops, so I'm for removing those too.



## Changes Proposed:
-  Remove guaranteed drops from Dalaran Brewmaster and Dalaran Miner loot tables

## Issues Addressed:

- Closes #6154
- Closes chromiecraft/chromiecraft#747

## SOURCE:

https://classic.wowhead.com/npc=3577
https://classic.wowhead.com/npc=3578
Sources like https://classicdb.ch/?npc=3578#drop claim these mobs *do* have guaranteed drops, but I deem wowhead to be more reliable

## Tests Performed:

- Inspected the new loot tables

## How to Test the Changes:


1. Inspect the new loot tables and make sure there are no guaranteed drops, or
2. Kill these NPCs in-game and make sure they don't have guaranteed drops


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
